hedera-node/src/main/java/com/hedera/services/ledger/properties/AccountProperty.java

```diff
@@ -78,10 +78,11 @@
 		public BiConsumer<MerkleAccount, Object> setter() {
 			return (a, v) -> {
 				try {
-					a.setBalance((long) v);
+					a.setBalance((long)v);
 				} catch (NegativeAccountBalanceException nabe) {
+					// The cast here is to make sonar happy
 					throw new IllegalArgumentException(String.format(
-							"Argument 'v=%d' would cause account 'a=%s' to have a negative balance!", v, a));
+							"Argument 'v=%d' would cause account 'a=%s' to have a negative balance!", (long)v, a));
```

Comment:
same here, should have the spacing `(long) v`

Reply:
Changed.